exchanges/hitbtc/hitbtc_wrapper.go

```diff
@@ -177,21 +177,23 @@ func (e *Exchange) Setup(exch *config.Exchange) error {
 }
 
 // FetchTradablePairs returns a list of the exchanges tradable pairs
-func (e *Exchange) FetchTradablePairs(ctx context.Context, _ asset.Item) (currency.Pairs, error) {
+func (e *Exchange) FetchTradablePairs(ctx context.Context, a asset.Item) (currency.Pairs, error) {
+	if a != asset.Spot {
+		return nil, fmt.Errorf("%w: %q", asset.ErrNotSupported, a)
+	}
```

Comment:
Please add a newline here just for clarity as everything else is grouped by purpose